application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -103,6 +114,8 @@ public ReminderCommand(Database database) {
     public void onSlashCommand(SlashCommandInteractionEvent event) {
         switch (event.getSubcommandName()) {
             case CREATE_SUBCOMMAND -> handleCreateCommand(event);
+            case CANCEL_COMMAND -> handleCancelCommand(event);
+            case CANCEL_ALL_COMMAND -> handleCancelAllCommand(event);
```

Comment:
personally, im not a fan of the `cancel all` command. id remove it 🤷

Reply:
done